recipes/pybufr-ecmwf/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.82" %}
+{% set sha256 = "3f8375f2ced161a203c3521520fdeb8b8b5b088987c98b609abdff7408b78a4c" %}
+
+package:
+  name: pybufr-ecmwf
+  version: {{ version }}
+
+source:
+  fn: pybufr-ecmwf-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pybufr-ecmwf/pybufr-ecmwf-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
```

Comment:
Which `toolchain` do you mean? I only found a conda-forge feedstock for toolchain but no explanation on how to use it.